### PR TITLE
Updates required for trio-docs after PR-314 is merged into Trio

### PR DESCRIPTION
**Pull request by marionbarker**
Includes Trio App Group information for Trio and xDrip4iOS
Includes warning that Glucose Direct does not work at this time.

### CODE REVIEW

File: docs/EN/operate/build.md [hunk 1]
```diff
@@ -67,6 +67,8 @@ After you have done this one-time step, each successive build will include the b
 
 You can add additional tests notes for any build if you desire.
 
+> End of Build Trio with GitHub
```
Comment (MikePlante1) [inline docs/EN/operate/build.md]: What is this line for? Here's how it appears on readthedocs:

screenshot

File: docs/EN/operate/build.md [hunk 2]
```diff
@@ -77,19 +79,38 @@ Second, make sure you have a valid Distributor certificate; if not, see [Distrib
...
 * [Apple Certificate Page](https://developer.apple.com/account/resources/certificates/list)
 
+### One-time Update for Trio App Group
+
+If you built previously, before the App Group changed from Loop to Trio, you need to do this step. Then you can continue with the updating instructions.
+
+* [Create the `Trio App Group`](#create-the-trio-app-group)
+* Examine the [Table of Identifiers](#table-of-identifiers) - it is optional, but editing the descriptions to match the **Names** in that table will make following directions easier in the future
+* Update the App Group to be the `Trio App Group` for the Identifiers (Trio, Trio Watch and Trio Watchkit Extension)
+* Select the Action: 3. Create Certificates to make certificate which go with the new `Trio App Group`
```
Comment (MikePlante1) [inline docs/EN/operate/build.md]: Maybe making **3. Create Certificates** bold would help readability?

File: docs/EN/operate/build.md [hunk 3]
```diff
@@ -77,19 +79,38 @@ Second, make sure you have a valid Distributor certificate; if not, see [Distrib
...
+
+### Consequences of `Trio App Group`
+
+If you use xDrip4iOS or GlucoseDirect as your CGM for Trio, they need to support the same `App Group` as Trio.
+
+At this time, 
+* xDrip4iOS requires version 5.3.1 to support the `Trio App Group`
```
Comment (MikePlante1) [inline docs/EN/operate/build.md]: I suggest adding ` or newer` after `5.3.1`

File: docs/EN/operate/build.md [hunk 4]
```diff
@@ -77,19 +79,38 @@ Second, make sure you have a valid Distributor certificate; if not, see [Distrib
...
+* GlucoseDirect has not been updated so will not work with Trio as a CGM source
```
Comment (MikePlante1) [inline docs/EN/operate/build.md]: Maybe append ` at this time` to the end of this line?

File: docs/EN/operate/build.md [hunk 5]
```diff
@@ -77,19 +79,38 @@ Second, make sure you have a valid Distributor certificate; if not, see [Distrib
 
 * [Apple Certificate Page](https://developer.apple.com/account/resources/certificates/list)
 
+### One-time Update for Trio App Group
+
+If you built previously, before the App Group changed from Loop to Trio, you need to do this step. Then you can continue with the updating instructions.
```
Comment (MikePlante1) [inline docs/EN/operate/build.md]: Maybe link `updating instructions` to #continue-updating ?

File: docs/EN/operate/build.md [hunk 6]
```diff
@@ -77,19 +79,38 @@ Second, make sure you have a valid Distributor certificate; if not, see [Distrib
...
+* GlucoseDirect has not been updated so will not work with Trio as a CGM source
+
+### Continue Updating
+
 Open your fork at `github.com/YOUR-USERNAME/Trio` with your GitHub username instead of `YOUR-USERNAME`
 
 * Look to see if your fork is up to date
 * If your fork shows that your branch is behind, sync the branch to get the latest updates
 
-Select Actions: 4. Build Trio and wait about an hour for your updated app to appear in TestFlight.
```
Comment (MikePlante1) [inline docs/EN/operate/build.md:85]: How come this was deleted? It seems to be an important step for updating Trio.

File: docs/EN/operate/build.md [hunk 7]
```diff
@@ -77,19 +79,38 @@ Second, make sure you have a valid Distributor certificate; if not, see [Distrib
...
-
-## Distribution Certificate
+### Distribution Certificate
 
 When you first build using GitHub, a Distribution Certificate is generated. It typically is valid for one year. You should get an email from Apple when you are within 30 days of that certificate expiring. When it expires, you can no longer build a new app. (Existing apps in TestFlight are not affected, they get the full 90 days.)
 
 The directions to renew your Distribution Certificate are found in [LoopDocs: Renew Certificate](https://loopkit.github.io/loopdocs/gh-actions/gh-update/#renew-certificate).
 
+> End of Update Trio with GitHub
```
Comment (MikePlante1) [inline docs/EN/operate/build.md]: Same question as the one on line 70.

File: docs/EN/operate/build.md [hunk 8]
```diff
@@ -196,6 +229,8 @@ Sometimes, specific branches are offered for testing. Any desired branch can be
   https://raw.githubusercontent.com/loopandlearn/lnl-scripts/main/BuildTrio.sh)" - branch_name
 ```
 
+> End of Build Trio with Script
```
Comment (MikePlante1) [inline docs/EN/operate/build.md]: Same question as the one on line 70.

File: docs/EN/operate/build.md [hunk 9]
```diff
@@ -231,42 +266,94 @@ git fetch
...
-* Return to the Trio folder: `cd ..`
-
-The Trio code is now updated.
+If there are no errors, the Trio code is now updated.
 
 If Xcode is not open, you can open it by typing `xed .` in the Trio folder of your terminal window.
 
 You can now build the updated Trio app on your phone.
 
+> End of Update Trio with Mac-Xcode
```
Comment (MikePlante1) [inline docs/EN/operate/build.md]: Same question as the one on line 70.

File: docs/EN/operate/build.md [hunk 10]
```diff
@@ -231,42 +266,94 @@ git fetch
...
-Otherwise, you are ready to build.
+The second command `git pull --recurse` does several things:
 
-### Fix Customization Conflicts
+1. Brings down any updates about the state of the submodule repositories from GitHub, e.g., `Fetching submodule XXXX` repeated for each submodule
+2. Merges any changes from GitHub into your local copy of the Trio workspace
+3. For each submodule, updates the version to match that selected in GitHub
 
-If you have customizations in the Trio code (in the workspace) that cause a conflict when you pull, you can save and restore those with the `stash` and `stash pop` lines as shown below:
+There can be errors after step 2, e.g.,
```
Comment (MikePlante1) [inline docs/EN/operate/build.md]: "step 2" and later "step 3" might be confusing to readers, thinking the should be doing three steps, when really all three steps are done in one command by the user.

I'm not sure how to word it better, though, so you can just mark this is "resolved" if nothing better comes to mind quickly.

File: docs/EN/operate/build.md [hunk 11]
```diff
@@ -231,42 +266,94 @@ git fetch
...
+	FreeAPS/folder/file.swift
+Please commit your changes or stash them before you merge.
+Aborting
 ```
-git stash
-git fetch
+
+The `pull` command halts at this error and does not move on to step 3. You must first fix the Trio workspace files. 
+
+Head over to [Fix Trio Conflicts](#fix-trio-conflicts).
```
Comment (MikePlante1) [inline docs/EN/operate/build.md]: I think merge this line into the previous line to get something like:

The `pull` command halts at this error and does not move on to step 3. You must first fix the Trio workspace files by following the instructions at [Fix Trio Conflicts](#fix-trio-conflicts).

File: docs/EN/operate/build.md [hunk 12]
```diff
@@ -231,42 +266,94 @@ git fetch
...
+
+* If there was no error with step 2, you will see the word `Updating` with the list of files changed.
+* If there were no updates, you'll see `Already up to date`
+
+### Submodule Update
+
+* If there are no updates to submodules, you won't see any submodule update messages - that is not uncommon
```
Comment (MikePlante1) [inline docs/EN/operate/build.md:297]: Missing `.` from the end. (I know, super nit-picky, feel free to disregard)

File: docs/EN/operate/build.md [hunk 13]
```diff
@@ -231,42 +266,94 @@ git fetch
...
+* If there are upstates to one or more submodules with no errors, you will see messages indicating the submodules are being checked out similar to the one below.
```
Comment (MikePlante1) [inline docs/EN/operate/build.md]: `upstates` should be `updates`

File: docs/EN/operate/build.md [hunk 14]
```diff
@@ -231,42 +266,94 @@ git fetch
...
+
+```{code-block}
+:class: no-copybutton
+Submodule path 'ModuleName': checked out 'f11abde5e2eea2cbf7ac80f3f4bc4bc6e7f6de56'
+```
+
+If you see the word error or conflict on the output, similar to the following, head over to [Fix Submodule Conflicts](#fix-submodule-conflicts).
```
Comment (MikePlante1) [inline docs/EN/operate/build.md]: Maybe put code blocks around `error` and `conflict`?

File: docs/EN/operate/build.md [hunk 15]
```diff
@@ -231,42 +266,94 @@ git fetch
...
-* Change directory to that module: `cd modulename`
-* Restore your changes: `git stash pop`
-* Return to the Trio folder: `cd ..`
-
-The Trio code is now updated.
+If there are no errors, the Trio code is now updated.
 
 If Xcode is not open, you can open it by typing `xed .` in the Trio folder of your terminal window.
 
 You can now build the updated Trio app on your phone.
```
Comment (MikePlante1) [inline docs/EN/operate/build.md]: Maybe `to your phone` instead of `on your phone`?

File: docs/EN/operate/build.md (hunk 1 above)
Comment (marionbarker) [reply docs/EN/operate/build.md]: The page jumps from topic to topic. I was thinking we needed some kind of marker that the reader was at the end of the particular topic. Easy to remove if you think it not useful. Or replace with a return to top of page link if you think that is better.

Let me know which is preferred.

File: docs/EN/operate/build.md (hunk 6 above)
Comment (marionbarker) [reply docs/EN/operate/build.md:85]: Thanks - I missed that - added back a little later (after Distribution Certificates)

File: docs/EN/operate/build.md (hunk 12 above)
Comment (marionbarker) [reply docs/EN/operate/build.md:297]: Bullets typically don't have periods. I removed a period from the second bullet.

**Comment by marionbarker** [comment]
Thanks for the detailed notes @MikePlante1 .
Ready for the next review.

**Review by MikePlante1 — APPROVED** [review]
LGTM. Thanks.

‼️ This merge needs to be coordinated with a message in Discord as well as the approval of https://github.com/nightscout/trio/pull/314 to the Docs. ‼️

**Comment by t1dude** [comment]
Looks good to me. Feel free to use me as docs approver as well.

**Pull request merged**